ts/model_server.py

```diff
@@ -48,6 +48,10 @@ def start() -> None:
             try:
                 parent = psutil.Process(pid)
                 parent.terminate()
+                try:
+                    os.remove(os.getcwd() + "/key_file.json")
+                except FileNotFoundError:
+                    print("Token authorization not enabled")
```

Comment:
You can print delete the file if the file exists.

Reply:
noted